scripts/ci/install.test

```diff
@@ -35,7 +35,11 @@ else
 fi
 
 # install the pre-built Bokeh package plus all test and runtime dependencies
-conda install -n testenv --yes --quiet --use-local bokeh `python scripts/deps.py run test` tornado=${TORNADO:-5}
+if [[ -z "${NUMPY}" ]]; then
+    conda install -n testenv --yes --quiet --use-local bokeh `python scripts/deps.py run test` tornado=${TORNADO:-6}
+else
+    conda install -n testenv --yes --quiet --use-local bokeh `python scripts/deps.py run test` tornado=${TORNADO:-6} numpy=${NUMPY}
```

Comment:
This could be coalesced to avoid repetition:
```sh
 conda install (...) tornado=${TORNADO:-6} numpy=${NUMPY:-*}
```

Reply:
will change